images/base/scripts/common/setup.sh

```diff
@@ -93,7 +85,7 @@ fi
 cd /tmp
 git clone https://github.com/google/protobuf.git
```

Comment:
Maybe for a future PR, but I think we could just download the release from https://github.com/google/protobuf/releases/download/v3.0.0-beta-3/protoc-3.0.0-beta-3-linux-x86_64.zip